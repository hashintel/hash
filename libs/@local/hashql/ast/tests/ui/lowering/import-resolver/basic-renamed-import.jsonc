//@ run: pass
//@ description: Tests that imports with aliases resolve correctly and the alias maps to the original item
["use", "core::bits", { "#struct": { "shl": "bit_shl" } }, "bit_shl"]
