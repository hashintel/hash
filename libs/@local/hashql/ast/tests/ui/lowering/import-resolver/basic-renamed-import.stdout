Expr#4294967040@20
  ExprKind (Path)
    Path#4294967040@20 (rooted: true)
      PathSegment#4294967040@19 (name: math)
      PathSegment#4294967040@19 (name: bit_shl)
